homework/list.js

```diff
@@ -1,44 +1,48 @@
 'use strict';
 
-(function() {
+(function(exports) {
+  
+  var TodoListManager = function() {
+    this._listNoteContent = [];
+    this._wrapper = document.querySelector('#note-list-wrapper');
+  };
+  
+TodoListManager.prototype = {
+  
+  start() {
+    this.fetchList((function(data) {
+      this.updateList(data);
+      this.drawList();
+      this.preloadFirstNote();
+    }).bind(this));
+    window.addEventListener(('click', function(event) {
+      this.onNoteOpen(event);
+    }).bind(this));
+  },
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
-
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+  onNoteOpen(event) {
```

Comment:
It's a convention to prefix method with `on` for the corresponding function, so it should be renamed as 'onClick'.